lib/MusicBrainz/Server/Edit.pm

```diff
@@ -341,7 +342,7 @@ sub TO_JSON {
         is_open => boolean_to_json($self->is_open),
         $can_preview ? (preview => boolean_to_json($self->preview)) : (),
         status => $self->status + 0,
-        votes => [map { $_->TO_JSON } $self->all_votes],
+        votes => to_json_array($self->votes),
```

Comment:
`all_votes` returns an array (i.e. list), `votes` returns an array reference. `to_json_array` works on references.

The reason I made `to_json_array` take an array ref and not an array is that it can handle cases where it receives a scalar value like `undef` (and just return that back). If it took an array, it would have no way to distinguish `undef` from a list of one element.